addons/common/XEH_postInit.sqf

```diff
@@ -233,6 +233,16 @@
 
 [QGVAR(setDate), {setDate _this}] call CBA_fnc_addEventHandler;
 
+[QGVAR(setUnitIdentity), {
+    params ["_unit", "_name", "_face", "_speaker", "_pitch", "_nameSound"];
+
+    _unit setName _name;
+    _unit setFace _face;
+    _unit setSpeaker _speaker;
+    _unit setPitch _pitch;
+    _unit setNameSound _nameSound;
```

Comment:
From Achilles and personal experiences, these should be a keybind option or a CBA option to copy with such depth. As some players may want to copy the load-outs and not make the entire group of clones.

Reply:
With this being a "deep copy", I would expect "everything" to be copied. I don't think a setting is necessary, however, this can always be re-assessed later based on feedback.

Reply:
It depends on the situation. I have, sometimes, experienced that I need only their gear not all of them to share voices, faces, etc. but, on the other hand, sometimes you just need a full clone (e.g. the specific officer was killed accidentally and you need to dupe him).

Reply:
I guess we could merge it for now and in a later update, devise a solution for this issue.